src/App.tsx

```diff
@@ -1,5 +1,30 @@
+import { Route, Routes } from 'react-router-dom';
+
+import AboutUs from './common/aboutUs/AboutUs';
+import AdminPage from './features/adminPage/AdminPage';
+import CartPage from './features/layout/cart/CartPage';
+import ContactPage from './common/contact/ContactPage';
+import HomePage from './features/homePage/HomePage';
+import Layout from './features/layout/Layout';
+import ProductSeriesPage from './features/products/productSeries/ProductSeriesPage';
+import UserPage from './features/userPage/UserPage';
+
 function App() {
-  return <h1 className="text-3xl font-bold underline">Hello world!</h1>;
+  return (
+    <div className="App">
+      <Routes>
+        <Route element={<Layout />}>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/aboutUs" element={<AboutUs />} />
```

Comment:
about-us, products or products-series, so you dont need uppercase

Reply:
fixed